src/ChipInput.js

```diff
@@ -575,6 +576,7 @@ class ChipInput extends React.Component {
           style={inputStyleMerged}
           dataSource={autoCompleteData}
           dataSourceConfig={dataSourceConfig}
+          listStyle={{ maxHeight: autocompleteHeight || '300px', overflow: 'scroll' }}
```

Comment:
The `300` should be specified in the `defaultProps` at the bottom, as default value for the `autocompleteHeight` prop. Also, the c of autoCompleteHeight should be capitalized.

That said, I think we'd be far more flexible if we allow the entire `listStyle` to be customized instead of hard-coding a height for the list. :thinking:

Reply:
Yeah, I was kind of thinking about that too actually... just allowing users to pass in their own styling fixes this issue, but also prevents any other issues that other people may have with autocomplete styling in the future (which I actually am already running into in the project I'm working on 😛). It's the more general and flexible approach, so I'm happy with that if you are.

Reply:
Would you mind updating the PR to add `autoCompleteListStyle` then? :wink: